pkg/secrets/k8s_secrets_storage/provide_conjur_secrets_test.go

```diff
@@ -383,3 +387,91 @@ func TestProvide(t *testing.T) {
 		}
 	}
 }
+
+func Test_secrets_contentChanges(t *testing.T) {
```

Comment:
Ruh-Roh, the name of this test function uses `snake_case`.  (Function name should be `TestSecretContentChanges`).

I fear we've been doing too many context switches between Bash and Go.